src/app/shared/components/capture-receipt/capture-receipt.component.ts

```
@@ -403,9 +403,9 @@ export class CaptureReceiptComponent implements OnInit, OnDestroy, AfterViewInit
     const isIos = this.devicePlatform === 'ios';
 
     const galleryPermissionName = isIos ? 'Photos' : 'Storage';
-    let title = 'Camera Permission';
+    let title = 'Camera permission';
     if (permissionType === 'GALLERY') {
-      title = galleryPermissionName + ' Permission';
+      title = galleryPermissionName + ' permission';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)